openedx.yaml

```diff
@@ -0,0 +1,19 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: frontend-app-learner
+owner: mattcarter
+
+oeps:
+  oep-2: true
```

Comment:
Nit: capitalize to `True`

Reply:
this appears resolved now.  @nsprenkle  do you agree?